src/HelpersServiceProvider.php

```diff
@@ -131,14 +116,36 @@ protected function extendRouter()
                 : RouteFacade::whereIn($param, $versions);
         };
 
-        Route::macro('versionRange', fn (VersionEnumContract $from, VersionEnumContract $to, $param = null) => $versionRange($from, $to, $param, $this));
-        Route::macro('versionFrom', fn (VersionEnumContract $from, $param = null) => $versionRange($from, null, $param, $this));
-        Route::macro('versionTo', fn (VersionEnumContract $to, $param = null) => $versionRange(null, $to, $param, $this));
-
-        RouteFacade::macro('versionRange', fn (VersionEnumContract $from, VersionEnumContract $to, string $param = null) => $versionRange($from, $to, $param));
-        RouteFacade::macro('versionFrom', fn (VersionEnumContract $from, $param = null) => $versionRange($from, null, $param));
-        RouteFacade::macro('versionTo', fn (VersionEnumContract $to, $param = null) => $versionRange(null, $to, $param));
-
-        RouteFacade::macro('version', fn (VersionEnumContract $version) => RouteFacade::prefix('v' . $version->value));
+        Route::macro('versionRange', fn (
```

Comment:
let's revert formatting, but try to short lines by making an alias for the VersionEnumContract - `Version`

Reply:
Done

Reply:
Done